### PR TITLE
Fix Issues Reported by CodeQL on the compiled Package

### PR DESCRIPTION
**Pull request by chensation**
Updated Highcharts to removed js/incomplete-sanitation issues.

Patched moment manually, as the issue is currently still present in the package.
See: https://github.com/moment/moment/issues/6237

### CODE REVIEW

File: src/SfxWeb/src/app/modules/charts/health-chart/health-chart.component.ts [hunk 1]
```diff
@@ -114,7 +114,6 @@ export class HealthChartComponent implements OnInit, AfterViewInit, OnChanges {
     this.options.tooltip.enabled = data.length === 3;
     (this.options.series[0] as SeriesPieOptions).data = data;
 
-    this.options.title.style.fontSize =  +this.width * .2 + 'pt';
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/charts/health-chart/health-chart.component.ts:117]: whats the reason for this change?

File: src/SfxWeb/src/app/modules/concurrent-events-visualization/rca-overview/rca-overview.component.ts [hunk 2]
```diff
@@ -1,174 +1,175 @@
...
-    return list;
-  }
-}
+import { AfterViewInit, Component, ElementRef, Input, OnChanges, OnInit, ViewChild } from '@angular/core';
+import { Chart, Options, chart, PointOptionsObject } from 'highcharts';
+import { IPregeneratedColor, pregeneratedColors } from 'src/app/Common/Constants';
+import { IConcurrentEvents } from 'src/app/Models/eventstore/rcaEngine';
+import { EventStoreUtils, ITimelineData, ITimelineItem } from 'src/app/Models/eventstore/timelineGenerators';
+import { Utils } from 'src/app/Utils/Utils';
+import { DataGroup } from 'vis-timeline/peer';
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/concurrent-events-visualization/rca-overview/rca-overview.component.ts:7]: are these two imports the only change here?

File: src/SfxWeb/src/app/modules/event-store/timeline/timeline.component.ts [hunk 3]
```diff
@@ -1,122 +1,123 @@
...
-  }
-}
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { TelemetryEventNames } from 'src/app/Common/Constants';
+import { mergeTimelineData } from 'src/app/Models/eventstore/periodicEventParser';
+import { ITimelineData, ITimelineItem, parseEventsGenerically } from 'src/app/Models/eventstore/timelineGenerators';
+import { DataService } from 'src/app/services/data.service';
+import { TelemetryService } from 'src/app/services/telemetry.service';
+import { TimelineGeneratorFactoryService } from 'src/app/services/timeline-generator-factory.service';
+import { DataGroup } from 'vis-timeline/peer';
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/event-store/timeline/timeline.component.ts:8]: is this the only change to the file? the rest is formatting?

**Review by jeffj6123 — APPROVED** [review]

File: src/SfxWeb/src/app/modules/charts/health-chart/health-chart.component.ts (hunk 1 above)
Comment (chensation) [reply src/SfxWeb/src/app/modules/charts/health-chart/health-chart.component.ts:117]: this is no longer a property of tile.style as of highcharts 11

**Pull request merged**